src/components/dashboard/components/dictionary/DictionaryCard.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import persistActiveconcept from './helperFunction';
 
-const DictionaryCard = (dictionary)=> {
+const DictionaryCard = dictionary => {
```

Comment:
This would normally throw a linting error, I don't know why you chose to omit the brackets.

Reply:
Done

Reply:
not yet